src/assets/css/_sass/elements/_page.scss

```diff
@@ -33,4 +31,40 @@ body {
 
 main {
   flex: 1 0 auto; // 3
-}
+  .hero {
```

Comment:
`.hero` and it's associated styles should be moved to it's own page css file and declared in the `main.scss`